src/dbcp/metadata/data_mart.py

```diff
@@ -46,6 +47,7 @@
     Column("infra_total_proposed_pm2_5_tonnes_per_year", Float),
     Column("battery_storage_existing_capacity_mw", Float),
     Column("battery_storage_existing_facility_count", Integer),
+    Column("battery_storage_existing_co2e_tonnes_per_year", Float),
```

Comment:
Can you explain where this new column is coming from? Presuming it's PUDL, but seems like it should get added to the data mart documentation?

Reply:
There is only one battery plant that has co2 emissions associated I think because it is colocated with a fossil fuel plant (plant_id_eia = 6377). TBH I'm not sure why it's popping up now given this battery and fossil generator have [been in the data for a while](https://data.catalyst.coop/pudl/core_eia923__monthly_generation_fuel?plant_id_eia__exact=6377). Maybe EIA assigned gave them the same plant id in this latest release? I would also expect there to be multiple battery + fossil generator plants. 

Given this is only one plant I don't think it's worth diving super deep. I'll add some docs!